content/en/docs/Overview/goals.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Design Goals"
+linkTitle: "Goals"
+weight: 1
+description: >
+  Reasons behind the Elekto design
+---
+
+Elekto was originally built to support the [Kubernetes Steering Committee elections](https://github.com/kubernetes/steering/blob/master/elections.md), supporting the existing workflow of that community.  It's also intended to avoid several chronic issues with usage of the [CIVS election system](https://civs1.civs.us/).
```

Comment:
```suggestion
Elekto was originally built to support the [Kubernetes Steering Committee elections](https://github.com/kubernetes/steering/blob/master/elections.md), supporting the existing workflow of that community.  It's also intended to avoid several chronic issues with the usage of the [CIVS election system](https://civs1.civs.us/).
```